docs/specs/offline_store_format.md

```diff
@@ -62,10 +62,10 @@ Here's how Feast types map to BigQuery types when using BigQuery for offline sto
 | Event Timestamp |   `DATETIME` |
 | BYTES | `BYTES` |
 | STRING | `STRING` |
-| INT32 | `INT64` |
+| INT32 | `INTEGER` |
 | INT64 | `INT64` |
 | DOUBLE | `FLOAT64` |
-| FLOAT | `FLOAT64` |
+| FLOAT | `FLOAT` |
```

Comment:
https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#integer_type INT64 and FLOAT64 seem like correct data type.

Reply:
@tedhtchang apologies, you're right. I've corrected my suggestion. Since 'INTEGER' and 'FLOAT' are seen in various places in BQ usage/ecosystem, it may be helpful to have these types explicitly written in the chart.

Reply:
I've also renamed the PR to be for a chart "update" rather than a "fix"